developing.adoc

```diff
@@ -65,8 +65,7 @@ To start integration tests, run:
 ```bash
 $ make integration BUNDLE_LOCATION=<bundle location> PULL_SECRET_FILE=<pull secret path>
 ```
-where `<bundle location>` is either the bundle's URL or its path in the filesystem,
-and `<pull secret path>` is the path to a file containing your OpenShift pull secret.
+where `<bundle location>` is either the bundle's URL, its path in the filesystem, or `embedded` if the bundle is embedded in the binary. The `<pull secret path>` is the path to a file containing your OpenShift pull secret.
```

Comment:
Have you considered not setting BUNDLE_LOCATION at all when the bundle is embedded?

Reply:
@cfergeau yeah, but I figured it's better explicit. If integration can be run without the argument, then it's not immediately obvious that sometimes BUNDLE_LOCATION is needed. Reminds me of `crc` binary being explicitly stated vs implicitly taken from top of the `PATH`.